client/landing/stepper/declarative-flow/test/site-migration-flow.tsx

```diff
@@ -18,6 +20,19 @@ const originalLocation = window.location;
 jest.mock( '../../utils/checkout' );
 jest.mock( '@automattic/data-stores/src/user/selectors' );
 jest.mock( 'calypso/landing/stepper/hooks/use-is-site-owner' );
+jest.mock( '../../utils/path', () => {
+	return {
+		getLoginUrl: jest.fn(
```

Comment:
It is required because the getLoginUrl() returns different values based on the feature flag, so when I mock the feature flags value it breaks the tests.